Workbooks/SapMonitor2.0/AIOpsInsights/Availability Insights/Auto RCA.workbook

```diff
@@ -68,15 +66,28 @@
             "isHiddenWhenLocked": true,
             "timeContext": {
               "durationMs": 86400000
-            }
+            },
+            "value": "SYSTEMDB"
```

Comment:
verify this one as well

Reply:
this does not affect the workbook as the value is always passed to it from parent template.